packages/mapviewer/src/utils/composables/useMovableElement.composable.js

```diff
@@ -14,8 +14,7 @@ const MovementSource = Object.freeze({
  * It is possible to specify custom offset with screen border, in order to avoid some part of the
  * app (such as the header) to be accessible to the movable element
  *
- * @param element Reference to a DOM element
- * @param options Options such as described below
+ * @param config {Object} Configuration object
  * @param {HTMLElement} [options.grabElement] HTML element on which the user has to grab to start
```

Comment:
we need to update the comment : 
- it's no longer `options.key` but `config.key`.
- also we need to add the `element` to the config doc here